src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -779,6 +782,10 @@ public function dumpString(Cursor $cursor, string $str, bool $bin, int $cut): vo
             $this->line .= $this->indentPad;
             $this->line .= \sprintf('<img src="data:%s;base64,%s" /></samp>', $cursor->attr['content-type'], base64_encode($cursor->attr['img-data']));
             $this->endValue($cursor);
+        } elseif (str_starts_with($str, 'http') && filter_var($str, FILTER_VALIDATE_URL)) {
```

Comment:
I find `http` to restrictive. What about `ftp://` or even `file://`? I think you should just use `FILTER_VALIDATE_URL` and make no assertion on the protocol being used